test/rack/prx_auth/token_data_test.rb

```diff
@@ -18,24 +18,20 @@
         'admin' => [456, 789, 1011]
       }
     })
-    assert token.authorized_resources['$'].nil?
-    assert token.authorized_resources['789'] == 'admin'
-    assert token.authorized_resources['123'] == 'member'
+    assert !token.authorized_resources.contains?('$')
```

Comment:
Okay, so looks like this supports the old `$ => ...` compression.  But doesn't introduce any new scope compression.  Do you anticipate we'll need something new to compress these when we get more scopes?

Reply:
yes, if we have as many scopes as I anticipate we will. I have a few thoughts about how to tackle that, can write them up if that seems like a good use of my time.

Reply:
+1.  If nothing else, a quick slack thread might be nice, just to record your crazy ideas for posterity.